e2e/utils/runtime-utils.ts

```diff
@@ -6,14 +6,9 @@ const Selector = {
 };
 
 export async function initializeRuntimeIfModalPresented(page: Page): Promise<void> {
-  let initializeButton: Button;
-  try {
-    initializeButton = new Button(page, Selector.runtimeInitializerCreateButton);
-    await initializeButton.waitUntilEnabled();
-  } catch (e) {
-    // Expected if the runtime already exists.
-    return;
+  const initializeButton = new Button(page, Selector.runtimeInitializerCreateButton);
+  const existsButton = await initializeButton.exists();
+  if (existsButton) {
+    await initializeButton.clickAndWait();
```

Comment:
Lesser time for the check: `exists()` max time is 2 seconds; `waitUntilEnabled()` max wait time is 30 seconds.

Reply:
It may take a few seconds for this modal to appear - the initial wait was intentional. If this is to workaround some slow interaction, we could instead consider racing this against another promise upstream in the test.

Reply:
I see. We can simply incr timeout like  `exists(10)` or higher? 30 sec is probably too long of wait for the modal to appear.

Reply:
The modal being shown is dependent on a network request completing which is why it takes several seconds. During manual testing I've seen this take ~5s so 10s may be too close. I really think racing a level up would be better. Could you describe the specific test and specific issue you're trying to mitigate?

Reply:
No issue related. It works. I thought it would be nice if playback time can be shortened even if time saving is small. I'll close the PR for now. Thank you.